src/Simplify.elm

```diff
@@ -973,8 +1021,8 @@ fromModuleToProject =
         )
 
 
-fromProjectToModule : Rule.ContextCreator ProjectContext ModuleContext
-fromProjectToModule =
+fromProjectToModule : Bool -> Rule.ContextCreator ProjectContext ModuleContext
```

Comment:
I'd just have this argument be `{ expectNaN : Bool }` or even `{ config_ | expectNaN : Bool }`